projects/core/src/cart/store/effects/cart.effect.ts

```diff
@@ -95,7 +110,9 @@ export class CartEffects {
     private productImageConverter: ProductImageConverterService,
     private occCartService: OccCartService,
     private cartData: CartDataService
-  ) {}
+  ) {
+    this.cartData.getDetails = true;
```

Comment:
we should probably pass this `getDetails` as `true` instead of setting it always to `true` here

Reply:
done